### DISCUSSION TITLE
Alternative column label for pannel type list

### DISCUSSION THREAD

**Discussion by marcoberri** [SigNoz/signoz#6129] [Q&A]
Hi,
is possible rename the name of column in pannel type list (only label)?

In json i found node "selectedTracesFields" but not exists "label" field for show alternative column's name

**Comment by srikanthccv** [comment]
No, currently it is not possible to have an alternative column label.

**Comment by srikanthccv** [comment]
Please follow in https://github.com/SigNoz/signoz/issues/6195
